firmware/Core/Src/telecommands/telecommand_executor.c

```diff
@@ -18,6 +21,18 @@ TCMD_parsed_tcmd_to_execute_t TCMD_agenda[TCMD_AGENDA_SIZE];
 ///         (i.e., filled with a not-yet-executed command).
 uint8_t TCMD_agenda_is_valid[TCMD_AGENDA_SIZE] = {0};
 
+/// @brief Converts a TCMD_TelecommandChannel_enum_t to a string representation.
+/// @param channel Input TCMD_TelecommandChannel_enum_t
+/// @return A pointer to a C-string representing the TCMD_TelecommandChannel_enum_t.
+const char* getTelecommandChannelName(TCMD_TelecommandChannel_enum_t channel) {
```

Comment:
Bad name format. Please use snake case, with proper prefix.

Great function to create though!

Reply:
Changed it to `telecommand_channel_enum_to_str` instead